### DISCUSSION TITLE
[제안] 봇 페이지의 부스트 버튼 관련

### DISCUSSION THREAD

**Discussion by ghost** [koreanbots/koreanbots#127] [제안] [answered]
## 기능 설명
> 부스터가 소유하고 있는 봇이라는 걸 띄워주는 버튼
> ![image](https://user-images.githubusercontent.com/46986283/92568282-eac6f380-f2b9-11ea-85b7-03a32b706a44.png)
> 을 클릭하면 부스터 혜택을 보여주는 페이지로 이동

## 필요한 이유
> 현재 이 버튼은 클릭이 되는 것 처럼 보이지만, 위에 올렸을 때 부스터가 소유하고 있다는 문구가 나온다는 것 빼고는 없습니다. 따라서 유저가 부스트 혜택이 어떤 건지 보려면 디스코드에 접속하고, 서버 내에서 부스터 혜택을 찾아야 한다는 것이 있습니다.
그래서 버튼에 부스터 혜택으로 이동하는 하이퍼링크를 달면 유저분들이 더 쉽게 부스터 혜택은 어떤 것인지 볼 수 있을 것입니 다.

## 기대 효과
> 유저 분들이 서버 부스트를 할 시 얻는 효과를 쉽게 확인할 수 있음.
> 위에 따라 더 많은 유저 분들이 부스트를 할 가능성이 높다라고 볼 수 있음.

## 확인
- [x] 중복되는 제안사항은 없나요?
- [x] 현실성 있는 제안인가요?

[a comment by an automated account is not shown]

**Comment by wonderlandpark** [comment]
v2 업데이트 때 적용하겠습니다

**Comment by wonderlandpark** [comment, marked as answer]
구현이 완료되었습니다.

해당 PR에서 확인해보세요: https://github.com/koreanbots/client/pull/49
